src/apis/entries_api.rs

```diff
@@ -74,25 +84,40 @@ pub async fn create_log_entry(configuration: &configuration::Configuration, prop
     let local_var_content = local_var_resp.text().await?;
 
     if !local_var_status.is_client_error() && !local_var_status.is_server_error() {
-        serde_json::from_str(&local_var_content).map_err(Error::from)
+        // Format the returned response such that it can be read into a struct
+        let uuid: &str = &local_var_content[1..67];
+        let rest: &str = &local_var_content[69..local_var_content.len() - 2];
+        let sum = "{\"uuid\": ".to_string() + uuid + "," + rest;
+        serde_json::from_str::<LogEntry>(&sum).map_err(Error::from)
```

Comment:
Could you make this parsing into its own function? I see you're doing it in several places, and if it was one function it would be easier to update later once we don't have the dynamic top-level key.